cmd/flux/create_source_oci.go

```diff
@@ -156,6 +158,15 @@ func createSourceOCIRepositoryCmdRun(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	if secretName := sourceOCIRepositoryArgs.verifySecretRef; secretName != "" {
+		repository.Spec.Verify = &sourcev1.OCIRepositoryVerification{
+			Provider: "cosign",
```

Comment:
We’ll support other signers, so we can’t hardcode cosign here. Let’s add a flag for `verify-provider` that must be specified if `verify-secret-ref` is provided.

Reply:
fixed, thanks!